src/app/EventLogging.h

```diff
@@ -69,6 +69,7 @@ class EventLogger : public EventLoggingDelegate
 template <typename T, std::enable_if_t<DataModel::IsFabricScoped<T>::value, bool> = true>
 CHIP_ERROR LogEvent(const T & aEventData, EndpointId aEndpoint, EventNumber & aEventNumber)
 {
+    assertChipStackLockedByCurrentThread();
```

Comment:
since both of these call `logMgmt.LogEvent`, would it be more generic to place the assert in `EventManagement::LogEventPrivate` ?

Reply:
done, thanks